
time      startTime
time      endTime
int32     maxTimeMiliSecs
int32     TxPower
TagData[] tagList
 
